ssd1306/ssd1306.py

```diff
@@ -185,3 +185,26 @@ def contrast(self, contrast):
     self.write_command(SETCONTRAST)
     self.write_command(contrast)
 
+  def set_start_end_cols(self, start_col=0, end_col=None):
+    if end_col is None:
+      end_col = self.columns - 1
+    if start_col < 0 or start_col > self.columns - 1:
+      raise ValueError('Start column must be between 0 and %d.' % (self.columns - 1,))
+    if end_col < 0 or end_col > self.columns -1:
+      raise ValueError('End column must be between 0 and %d.' % (self.columns - 1,))
+
```

Comment:
Is it possible to add another check to make sure that ```start_col``` is smaller than ```end_col```, or is this something that is possible? If that would be the case, what's the behavior?

The same for ```start_page``` and ```end_page``` below.